src/Protos/Electricity-primitives.proto

```diff
@@ -0,0 +1,42 @@
+syntax = "proto3";
+
+import "Primitives.proto";
+
+option csharp_namespace = "ProjectOrigin.Electricity";
+
+package ProjectOrigin.Electricity.v1;
+
+message Slice {
```

Comment:
Very nice interface for the slice it ties perfectly to the Pedersen commitment scheme